whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageService.java

```diff
@@ -53,7 +54,13 @@ public void verifyAndSetAsUndeliverable(final EmailMessageInfo message){
         }
 
         LOGGER.debug("Undeliverable message-id {} email {}", message.messageId(), StringUtils.join(message.emailAddresses(), ", "));
-        message.emailAddresses().forEach(email -> emailStatusDao.createEmailStatus(email, EmailStatus.UNDELIVERABLE));
+        message.emailAddresses().forEach(emails -> {
+            try {
+                message.emailAddresses().forEach(email -> emailStatusDao.createEmailStatus(email, EmailStatus.UNDELIVERABLE));
```

Comment:
Its a nested FOR loop. Can you create individual email status when you catch exception inside a FOR loop?

Reply:
Copy paste issue, I was looping twice. It is not needed to loop twice.